css-canvas-medical/styles.css

```diff
@@ -0,0 +1,307 @@
+.container {
+  width: 100%;
+  margin: auto;
+  max-width: 1200px;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+}
+
+.row5 {
+  padding-top: 50px;
+}
+
+.img1-canvas {
+  margin-top: 30px;
+}
+
+.row-1 {
+  flex-direction: column;
+  width: 100%;
+  padding-top: 50px;
+  padding-left: 50px;
+}
```

Comment:
There should not be variants of the `row` class. This styling doesn't seem related to the functionality of the `row` class anyway, you should rename this class to describe its purpose.